.vscode/settings.json

```diff
@@ -1,21 +1,15 @@
 {
   "[ruby]": {
     "editor.autoClosingBrackets": "beforeWhitespace",
+    "editor.formatOnType": true,
+    "editor.insertSpaces": true,
+    "files.insertFinalNewline": true,
+    "editor.semanticHighlighting.enabled": true,
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "rebornix.ruby"
+    "editor.defaultFormatter": "Shopify.ruby-lsp"
   },
   "files.trimTrailingWhitespace": true,
   "files.trimFinalNewlines": true,
   "editor.formatOnSave": true,
   "yaml.format.enable": true,
-  "ruby.useBundler": true, //run non-lint commands with bundle exec
-  "ruby.useLanguageServer": true, // use the internal language server (see below)
-  "ruby.lint": {
-    "rubocop": {
-      "useBundler": true,
-      "lint": true, // enable lint cops
-      "rails": true // requires rubocop-rails gem for RuboCop >= 0.72.0
-    }
-  },
-  "ruby.format": "rubocop",
```

Comment:
Should I be looking for a similar configuration for `ruby-lsp`

Reply:
Not right now. We can do that as next step. Right now lets get the bare essentials out to public.